pytorch_lightning/accelerators/accelerator.py

```diff
@@ -158,12 +161,12 @@ def on_train_start(self) -> None:
 
     def training_step(
         self,
-        args: List[Union[Any, int]],
+        kwargs: Dict[str, Union[Any, int]],
```

Comment:
Does this work when the batch is a dict ?

Reply:
yes, it does. Everything is the same, it's just that previously it was a tuple (batch, batch_idx, ...) and not it is an OrderedDict {'batch': batch, 'batch_idx': batch_idx}.